roles/edpm_nova/molecule/default/verify.yml

```diff
@@ -43,6 +43,12 @@
         - "Copying /var/lib/kolla/config_files/02-nova-log.conf to /etc/nova/nova.conf.d/02-nova-log.conf"
         - "Copying /var/lib/kolla/config_files/ssh-config to /var/lib/nova/.ssh/config"
         - "Copying /var/lib/kolla/config_files/ssh-privatekey to /var/lib/nova/.ssh/ssh-privatekey"
+        - "Copying /var/lib/kolla/config_files/02-nova-host-specific.conf to /etc/nova/nova.conf.d/02-nova-host-specific.conf"
+
+    - name: Assert that my_ip is rendered into the host specific config
+      ansible.builtin.assert:
+        that:
+          - "'10.0.0.3' in lookup('ansible.builtin.file', '/var/lib/openstack/config/nova/02-nova-host-specific.conf')"
```

Comment:
ack so this was executed in ci correctly

https://review.rdoproject.org/zuul/build/b2cdfbf23ae648a6a902c7cd8f383c73/log/ansible-execution.log#1301-1303